homeassistant/components/osoenergy/config_flow.py

```diff
@@ -22,7 +22,6 @@ class OSOEnergyFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     """Handle a OSO Energy config flow."""
 
     VERSION = 1
-    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
```

Comment:
Please remove this change from this PR

Reply:
removed

Reply:
new PR: https://github.com/home-assistant/core/pull/107553